wordfreq/__init__.py

```diff
@@ -123,12 +122,15 @@ def get_frequency_list(lang, wordlist='combined', match_cutoff=30):
             "nearest match, which is %r (%s)."
             % (lang, best, langcodes.get(best).language_name('en'))
         )
-
-    filepath = available[str(best)]
-    return read_dBpack(str(filepath))
+        
+    return read_dBpack(available[best])
 
 
 def dB_to_freq(dB):
+    """
+    Converts decibels to freequency.
+    If the decibels is positive, throws a ValueError
```

Comment:
"frequency" is misspelled, and this docstring doesn't tell you much beyond the title of the function.

I don't think it makes sense to just say "If the decibels is positive, throws a ValueError". This makes it sound like some sort of unexpected exception. If the docstring describes how we're expressing frequencies on a decibel scale, it could say that 0 dB is the maximum possible frequency. Then you wouldn't have to warn people about ValueErrors because it would be clear that frequencies above 0 dB are nonsensical.